resources/qml/Menus/ConfigurationMenu/ConfigurationMenu.qml

```diff
@@ -65,6 +70,7 @@ Cura.ExpandablePopup
                         id: extruderIcon
                         materialColor: model.color
                         extruderEnabled: model.enabled
+                        anchors.horizontalCenter: parent.horizontalCenter
```

Comment:
Is this property really needed? With this property the alignment of the extruders is incorrect.
![Screenshot 2022-05-11 at 10 45 50](https://user-images.githubusercontent.com/6638028/167810949-36dc980a-bd31-479f-941c-f2219e79b81b.png)
When this property is removed the alignment is correct again

Reply:
https://github.com/Ultimaker/Cura/pull/12161/commits/f0d3686ee7dc9dac9fad6bb6786587efdf21b752